doc/authentication-methods/client-certificate.md

```diff
@@ -46,11 +46,33 @@ Below you may find a list of backends that are safe to use with `cyrsasl_externa
 * `rdbms`
 * `riak`
 
+### Self-signed certificates
+
+By default MongooseIM doesn't accept self-signed certs for the SASL-EXTERNAL authentication.
+They may be handy in development so we made it possible to tell MongooseIM to accept them.
+
+#### Self-signed certificates for regular TCP/TLS connections
+
+In order to tell MongooseIM to accept self-signed certs, `ssl_options` list needs to be added to `ejabberd_c2s` listener config like below:
```

Comment:
In order to tell MongooseIM to accept self-signed certs, the `ssl_options` list needs to be added to `ejabberd_c2s` listener config like below: